### PR TITLE
Feature/look at when no c mesh fix

### PR DESCRIPTION
**Pull request by AlfonsoMestres**
Duplicated branch cause Case sensitive stuff. @AniolAndres

### CODE REVIEW

File: Source/ComponentCamera.cpp [hunk 1]
```diff
@@ -187,16 +187,23 @@ void ComponentCamera::Center()
...
+	if (App->scene->selected->GetComponent(ComponentType::Renderer) != nullptr)
+	{
+		AABB bbox = App->scene->selected->GetBoundingBox();
+		float3 HalfSize = bbox.HalfSize();
+		float distX = HalfSize.x / tanf(frustum->horizontalFov*0.5f);
+		float distY = HalfSize.y / tanf(frustum->verticalFov*0.5f);
+		float camDist = MAX(distX, distY) + HalfSize.z; //camera distance from model
+
+		float3 center = bbox.FaceCenterPoint(5);
+		frustum->pos = center + float3(0, 0, camDist);
```
Comment (AlfonsoMestres) [inline Source/ComponentCamera.cpp]: floats should end with f, watchout!

File: Source/ComponentCamera.cpp [hunk 2]
```diff
@@ -187,16 +187,23 @@ void ComponentCamera::Center()
...
+	}
+	else
+	{
+		float camDist = 50;
```
Comment (AlfonsoMestres) [inline Source/ComponentCamera.cpp]: same

File: Source/ComponentCamera.cpp [hunk 3]
```diff
@@ -196,13 +196,13 @@ void ComponentCamera::Center()
 		float camDist = MAX(distX, distY) + HalfSize.z; //camera distance from model
 
 		float3 center = bbox.FaceCenterPoint(5);
-		frustum->pos = center + float3(0, 0, camDist);
+		frustum->pos = center + float3(0.0f, 0.0f, camDist);
```
Comment (AlfonsoMestres) [inline Source/ComponentCamera.cpp]: Fixed!

**Comment by solidajenjo** [comment]
Why did you deleted the settextures part?

File: Source/MaterialEditor.cpp [hunk 4]
```diff
@@ -172,15 +172,6 @@ void MaterialEditor::TextureSelector(unsigned i, std::string &current_texture)
 		ImGui::Image((ImTextureID)material->textures[i]->id, { 200,200 }, { 0,1 }, { 1,0 });
 	}
 }
-
-void MaterialEditor::SetCurrentTextures()
-{
-	// Get material textures
-	Texture* diffuse_texture = material->GetTexture(TextureType::DIFFUSE);
```
Comment (AlfonsoMestres) [inline Source/MaterialEditor.cpp]: @AniolAndres, @solidajenjo  is right. Missed this one <:'D

Comment (AniolAndres) [reply Source/MaterialEditor.cpp]: Must've been a mistake, thanks for pointing it out. It was duplicated so one was erased, check branch for prove.

**Comment by solidajenjo** [comment]
Ok sorry, it looked a little weird when textures don't have any relation with your task

**Pull request merged**